pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -342,7 +342,9 @@ public boolean isGlobal() {
     public String getSchemaName() {
         return getTenant()
             + "/" + getNamespacePortion()
-            + "/" + getEncodedLocalName();
+            + "/" + (isPartitioned() ?
```

Comment:
getPartitionedTopicName already implemented the functionality

Reply:
Ok, I could reuse it.